services/js-poc/deno_fleek/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "deno_fleek"
+version = "0.0.1"
+edition.workspace = true
+
+[dependencies]
+anyhow.workspace = true
+arrayref = "0.3"
+blake3-tree = { path = "../../../lib/blake3-tree" }
+cid = "0.11"
+fn-sdk = { path = "../../../lib/sdk" }
+fleek-crypto.workspace = true
+lightning-schema = { path = "../../../core/schema" }
+lightning-workspace-hack.workspace = true
+log = "0.4"
```

Comment:
why log over tracing? We already use tracing in the service itself, as well as the node

Reply:
No real reason, I was copying and pasting and getting the core of the solution implemented...as mentioned before this is a draft.